src/estimation-kdl/CMakeLists.txt

```diff
@@ -29,6 +29,10 @@ target_link_libraries(${libraryname} idyntree-core idyntree-kdl idyntree-sensors
 get_property(IDYNTREE_TREE_INCLUDE_DIRS GLOBAL PROPERTY IDYNTREE_TREE_INCLUDE_DIRS)
 target_include_directories(${libraryname} BEFORE PRIVATE ${IDYNTREE_TREE_INCLUDE_DIRS})
 
+if((${CMAKE_CXX_COMPILER_ID} MATCHES "GNU") OR (${CMAKE_CXX_COMPILER_ID} MATCHES "Clang"))
+    target_compile_options(${libraryname} PRIVATE "-Wno-deprecated")
```

Comment:
I just realized that we need to support CMake 3.0 (at least for the next release, see http://wiki.icub.org/wiki/YARP_Supported_Distributions#Policy). As this option is not available in CMake 3.0, I think we should add a condition on the CMake version in the `if`.